src/features/pages/partners/deriv-prime/potential/data.tsx

```diff
@@ -0,0 +1,41 @@
+import BrokerageFirms from 'images/svg/deriv-prime/brokerage_firms.svg'
+import InvestmentFunds from 'images/svg/deriv-prime/investment_funds.svg'
+import AssetManagers from 'images/svg/deriv-prime/asset_managers.svg'
+import ProfessionalTraders from 'images/svg/deriv-prime/professional_traders.svg'
+import FinancialInstitutions from 'images/svg/deriv-prime/financial_institutions.svg'
+import CorporateClients from 'images/svg/deriv-prime/corporate_clients.svg'
+import { BoxData } from 'features/components/molecules/shadow-box'
+
+export const boxData: BoxData[] = [
```

Comment:
Data name should be `snake_case` ..`box_data`

Reply:
Hi @aswathy-deriv  I have fixed this 🙏